OpenEXR/IlmImf/ImfDeepTiledInputFile.cpp

```diff
@@ -605,7 +605,7 @@ TileBufferTask::execute ()
 	// sanity check data size: the uncompressed data should be exactly 
 	// 'sizeOfTile' (if it's less, the file is corrupt and there'll be a buffer overrun)
 	//
-        if(_tileBuffer->dataSize != sizeOfTile)
+        if(_tileBuffer->dataSize != Int64(sizeOfTile))
```

Comment:
I think we are preferring c++ style casts, in this case static_cast, in OpenEXR to C style cast or conversion constructors.

Reply:
I was going to use static_cast but... that Int64() was used elsewhere in the file... so let me know if you want be to switch those casts over too... it would mean more changed lines... but the casting style would be consistent within the file.

Reply:
ok done.